st2client/st2client/models/core.py

```diff
@@ -436,7 +436,21 @@ def resume(self, execution_id, **kwargs):
 
 
 class InquiryResourceManager(ResourceManager):
-    pass
+
+    @add_auth_token_to_kwargs_from_env
+    def respond(self, inquiry_id, inquiry_response, **kwargs):
+        """
+        Update st2.inquiry.respond action
+        Update st2client respond command to use this?
+        """
+        url = '/%s/%s' % (self.resource.get_url_path_name(), inquiry_id)
+
+        resp = self.client.put(url, inquiry_response, **kwargs)
```

Comment:
minor s/resp/response to be consistent.

Reply:
Ack. FWIW I did this because it was a little confusing with the `inquiry_response`, but an attentive eye should catch the difference anyways. Fixed in b15be5308